pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dto/TrainerDTO.java

```diff
@@ -12,6 +12,8 @@
  */
 public class TrainerDTO {
```

Comment:
Regenerate the toString, equals and hashCode methods in order to have id there to.